setup.py

```diff
@@ -39,7 +39,7 @@ def collectExtraFiles():
 setup(
     name="armi",
     # duplicating with meta.py for now. See comments there for rationale.
-    version="0.1.6",
+    version="0.1.7",
```

Comment:
we should update the release notes with a version bump.

Reply:
Okay. I notice this file `armi/doc/release/0.1.rst` already has a placeholder section for the `0.1.7` release (I hadn't noticed this folder before).

I assume the best thing to do, would be to just add the `localization` change to that last, and fix the dates?

Or are there other important changes we want to not there?